expan/core/experiment.py

```diff
@@ -141,14 +134,13 @@ def _delta(self, method, data, **worker_args):
                     power = statx.compute_statistical_power(treatment_data, control_data)
                     statistics['statistical_power'] = power
                 if len(w):
-                    result['warnings'].append('kpi: ' + kpi + ', variant: '+ variant + ': ' + str(w[-1].message))
-                res_kpi['variants'].append({'name': variant, 'delta_statistics' : statistics})
+                    result['warnings'].append('kpi: {}, variant: {}: {}'.format(kpi, variant, w[-1].message))
```

Comment:
do we have a standard on that?

Reply:
this is just simpler version of the concatentation. Here you don't need to do type casting implicitly.

Reply:
Oh, you're right, I thought you were constructing a `dict` instead of a `str`, sorry

Reply:
no problem!